### PR TITLE
Signup Page

### PR DESCRIPTION
**Pull request by rahul0396**
https://user-images.githubusercontent.com/43301962/154840027-d91862b9-d502-41c3-a4c1-e9b2dbb057bf.mp4

### CODE REVIEW

File: css/login.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+@import url("/css/navbar.css");
+
+body{
+    background-color: #F7F7F7;
+}
+.login-box{
+    position: absolute;
+    /* width: 500px;
```
Comment (RakshithBhat03) [inline css/login.css:8]: You can remove these comments

File: css/login.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    height: 350px; */
+    left: 30%;
+    top: 20%;
+    /* background-color: white; */
+
+}
+.a-remember-me{
+    margin-left: 6rem;
+    color: rgb(77, 77, 243);
```
Comment (RakshithBhat03) [inline css/login.css:17]: You can use a variable here

File: pages/signup.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+   <!-- login box start -->
+   <div class="login-box">
+       <div class="heading">
+           
+    </div>
+    
+    <form class="form">
+            <h1 class="card-heading">Sign up</h1>
+           <!-- <label class="input-label"for="">Email address</label> -->
```
Comment (RakshithBhat03) [inline pages/signup.html:60]: You can remove this comment

File: pages/signup.html [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+         name="username"
+         placeholder="Enter Your Last Name"
+       />
+           <input
+              class="input"
+              type="email"
+              name="email-id"
+              placeholder="Enter Your Email"
+            />
+            <!-- <label for="">Password</label> -->
```
Comment (RakshithBhat03) [inline pages/signup.html:79]: Same here as well

File: pages/signup.html [hunk 5]
```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Signup Page</title>
+</head>
+<!-- stackzer-ui css file -->
+<link rel="stylesheet" href="https://stackzer.netlify.app/css/main.css"/>
```
Comment (RakshithBhat03) [inline pages/signup.html:10]: Add the stylesheet inside the `<head>` tag.

File: pages/signup.html [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Signup Page</title>
+</head>
+<!-- stackzer-ui css file -->
+<link rel="stylesheet" href="https://stackzer.netlify.app/css/main.css"/> 
+
+
+<link rel="stylesheet" href="/css/login.css">
```
Comment (RakshithBhat03) [inline pages/signup.html:13]: Same here as well

**Review by RakshithBhat03 — COMMENTED** [review]
Suggested a few changes. You can merge once you have implemented the changes.

**Review by ZuberDunge — APPROVED** [review]
Apart from what Rakshit said, Rest looks good to me.

**Pull request merged**